src/common/server.ts

```diff
@@ -44,18 +45,45 @@ export type IInitializationOptions = {
   globalSettings: ISettings;
 };
 
+/**
+ * Check if shell mode is required for execFile.
+ * Note: This is only the case for Windows OS when using .cmd instead of .exe extension to start the interpreter.
+ * @param file
+ * @returns
+ */
```

Comment:
Do you mind removing the `@param` and `@return` from the doc? I don't think it provides any additional information and the arguments are already annotated with types.